README.md

```diff
@@ -82,6 +82,14 @@ npm install --save react-a11y-dialog
 
 ---
 
+- **Property name**: `closeButtonPosition`
+- **Type**: string
+- **Mandatory**: false
+- **Default value**: `first`
```

Comment:
Shall we go uppercase for these values?

Reply:
I would not do that as it goes against common practice for naming properties.

Reply:
Does it? Why?

Reply:
Ha, common practice was maybe a bit of a stretch :) I am just referring to consistent naming and in my experience JSX properties still try to be consistent with HTML values and those are all lowercase. It would strike me as odd if suddenly these API properties are in uppercase; never seen it in any React library. What do you think?

Reply:
Actually that’s a solid point, I like it. 👍